src/systems/filecoin_vm/interpreter/_index.md

```diff
@@ -26,7 +26,8 @@ For each tipset:
 
 - the `CronActor`'s tick method is invoked implicitly after all the blocks' messages.
 
-These implicit messages specify a gas price of zero, but must be included in the computation.
+These implicit messages have a `From` address being the distinguished system account actor.
+They specify a gas price of zero, but must be included in the computation.
```

Comment:
The decision was made to make them free? How do we deal with overflows or long-running computation from cron (for future PR I reckon)

Reply:
Yes. Overflows are not handled yet (the gas limit is really, really big but we can probably do better - add it to the list of parameters TBD).